pkg/oc/cli/cmd/set/deployment_strategy.go

```diff
@@ -0,0 +1,483 @@
+package set
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/kubernetes/pkg/apis/apps"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+
+	deployapi "github.com/openshift/origin/pkg/apps/apis/apps"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+var (
+	deploymentStrategyLong = templates.LongDesc(`
+		Set a deployment strategy for a deployment config, deployment,
+		statefulset and daemonset.
+
+`)
+
+	deploymentStrategyExample = templates.Examples(`
+		# Change the deployment strategy for dc/myapp to recreate
```

Comment:
Will the indentation here get fixed automatically?

Reply:
i think it does.